mlflow/server/handlers.py

```diff
@@ -640,18 +640,31 @@ def wrapper(*args, **kwargs):
 @catch_mlflow_exception
 def get_artifact_handler():
     run_id = request.args.get("run_id") or request.args.get("run_uuid")
+    model_id = request.args.get("model_id")
```

Comment:
can we create a separate endpoint for logged model artifacts?

Reply:
We could, but why do we want to have a separate endpoint?

Reply:
btw we already have `get_logged_model_artifact_handler`.

Reply:
In my understanding, it's normal for an endpoint to allow callers to specifly a resource by several different parameter sets (in the layered architecture or clean architecture we usually have parameter handling layer). 
It's a bit weird interface if general /get-artifact endpoint can only serve run artifacts.

Reply:
But it seems we already have the endpoint for logged model artifact as you mentioned, let me handle these in the frontend.

Reply:
@harupy Can you take another look?